chatmaild/src/chatmaild/filtermail.py

```diff
@@ -14,6 +14,64 @@
 
 from .config import read_config
 
+"""Generated from deltachat, draft-ietf-lamps-header-protection, and
+encrypted_subject localizations in
+https://github.com/thunderbird/thunderbird-android/
+"""
+common_encrypted_subjects = {
+    "...",
+    "[...]",
+    "암호화된 메시지",
+    "Ĉifrita mesaĝo",
+    "Courriel chiffré",
+    "Dulrituð skilaboð",
+    "Encrypted Message",
+    "Fersifere berjocht",
+    "Kemennadenn enrineget",
+    "Krüptitud kiri",
+    "Krypterat meddelande",
+    "Krypteret besked",
+    "Kryptert melding",
+    "Mensagem criptografada",
+    "Mensagem encriptada",
+    "Mensaje cifrado",
+    "Mensaxe cifrada",
+    "Mesaj Criptat",
+    "Mesazh i Fshehtëzuar",
+    "Messaggio criptato",
+    "Messaghju cifratu",
+    "Missatge encriptat",
+    "Neges wedi'i Hamgryptio",
+    "Pesan terenkripsi",
+    "Salattu viesti",
+    "Şifreli İleti",
+    "Šifrēta ziņa",
+    "Šifrirana poruka",
+    "Šifrirano sporočilo",
+    "Šifruotas laiškas",
+    "Tin nhắn được mã hóa",
+    "Titkosított üzenet",
+    "Verschlüsselte Nachricht",
+    "Versleuteld bericht",
+    "Zašifrovaná zpráva",
+    "Zaszyfrowana wiadomość",
+    "Zifratu mezua",
+    "Κρυπτογραφημένο μήνυμα",
+    "Зашифроване повідомлення",
+    "Зашифрованное сообщение",
+    "Зашыфраваны ліст",
+    "Криптирано съобщение",
+    "Шифрована порука",
+    "დაშიფრული წერილი",
+    "הודעה מוצפנת",
+    "پیام رمزنگاری‌شده",
+    "رسالة مشفّرة",
+    "എൻക്രിപ്റ്റുചെയ്‌ത സന്ദേശം",
+    "加密邮件",
+    "已加密的訊息",
+    "暗号化されたメッセージ",
+}
+
```

Comment:
hum, we decided against going for all of these strings at one point. 

Shouldn't K-9 mail rather change to `[...]`?

Reply:
the thing is that a localized "encrypted subject" 

a) leaks the mail sender's locale 
b) mail receiver locale/language might not be able to make sense of sender's localized "encrypted subject"

Reply:
yes, k-9 should change, for the reasons you describe and more. i've asked them to do that at https://github.com/thunderbird/thunderbird-android/issues/8011

However, even if they adopt that change, existing deployments won't change until there is an upgrade path and it actually happens.

If chatmail wants to be interoperable with the deployed base, it should also change, as described here.

Once there are widely-deployed instances of k9 and thunderbird-android that do the sensible standardized thing, then maybe chatmail could consider reverting this.  But in the meantime, this seems like a reasonable measure for interoperability.